java/org/contikios/cooja/plugins/TimeLine.java

```diff
@@ -124,7 +124,9 @@ public class TimeLine extends VisPlugin implements HasQuickHelp {
   private static final long[] ZOOM_LEVELS = {
   	1, 2, 5, 10,
   	20, 50, 100, 200, 500, 1000,
-  	2000, 5000, 10000, 20000, 50000, 100000 };
+  	2000, 5000, 10000, 20000, 50000, 100000, 
+  	100000*2, 100000*5, 100000*10
```

Comment:
Is the crash related to zoom levels not containing enough values, or is this change for usability?